test/runtests.jl

```diff
@@ -64,11 +72,13 @@ cd(dirname(@__FILE__)) do
         error("Some tests exited with errors.")
     end
 
-    if compile_test
+    # Free up memory =)
+    n > 1 && rmprocs(workers(), waitfor=5.0)
+
+    for t in node1_tests
         n > 1 && print("\tFrom worker 1:\t")
-        runtests("compile")
+        runtests(t)
```

Comment:
Can probably shave off another couple of tens of seconds by running this in parallel upto CPU_CORES. While the `compile` test will not yield, the `parallel` test spawns out its test script and would not block pid 1.